browser-test/src/admin_predicates.test.ts

```diff
@@ -314,10 +317,62 @@ describe('create and edit predicates', () => {
       await logout(page)
     })
 
+    it('multiple values and multiple questions', async () => {
```

Comment:
Also need a test for editing an existing condition both displaying the existing condition and saving the new one..  eligibility and visibility

Reply:
done